docker-for-mac/index.md

```diff
@@ -462,6 +462,28 @@ ln -s $etc/docker.zsh-completion /usr/local/share/zsh/site-functions/_docker
 ln -s $etc/docker-compose.zsh-completion /usr/local/share/zsh/site-functions/_docker-compose
 ```
 
+### Fish-Shell
+
+Fish-shell also supports tab completion [completion
+system](https://fishshell.com/docs/current/#tab-completion){:target="_blank"
+class="_"}. To activate completion for Docker commands,
+these files need to be copied or symlinked to your Fish-shell `completions/`
+directory.
+
+Create at first the `completions` directory if not exist:
```

Comment:
"at first" looks a bit redundant here. With my suggestion below (to add the `-p` option), I think we can simplify the sentence, and remove the "if not exist";

```suggestion
Create the `completions` directory:
```

Alternatively, start with "first";

```suggestion
First, create the `completions` directory:
```